### ISSUE TITLE
Question about FOV (tile) y coordinate

### ISSUE THREAD

**Issue by lambdamoses** [labsyspharm/ashlar#88]
Thank you for this awesome tool! I have tried it with the mcmicro exemplar, and tried it on my own dataset. I could either specify the FOV locations in a grid (I saw that in your source code), or resave my dataset into OME-TIFF with FOV coordinates in the metadata. It seems that ashlar expects a certain orientation of the y FOV coordinate. So when I used BigStitcher on the mcmicro exemplar, the y FOV coordinates are flipped so the tile configuration does not display properly as small values of y is near the bottom while BigStitcher expects large values of y to be near the bottom. Conversely, for my OME-TIFF dataset, which was successfully stitched with BigStitcher, ashlar gave bad results. The grid method still worked. So for Bio-Formats metadata, is there a way to specify which way the y FOV coordinate should go?

**Comment by Yu-AnChen** [comment]
Thanks for trying ashlar out 🙂 We are aware of the issue stage coordinates and image flipping, each microscope/camera vendor can have a different orientation based on their applications. The microscope we used the most happens to have a different y stage coordinate, as compared to some other vendors like Zeiss, that requires a flip which is the default in ashlar.  

It sounds like you are working in a python session, if the stage position metadata and image pixel size in your input file (OME-TIFF) is recorded correctly, this is what I would do 

```python
import ashlar.reg
import ashlar.scripts.ashlar as ashlar_scripts

reader = ashlar.reg.BioformatsReader('path_to_input_ome_tiff')
ashlar_scripts.process_axis_flip(reader=reader, flip_x=False, flip_y=True)

# construct stitching object after flipping axis
# pass other kwargs as needed
edge_aligner = ashlar.reg.EdgeAligner(reader)
edge_aligner.run()
```

If you are working in the command-line interface, you can pass the `--flip-y` flag to achieve the `ashlar_scripts.process_axis_flip(reader=reader, flip_x=False, flip_y=True)` mentioned above

```bash
ashlar "path_to_input_ome_tiff" --flip-y
```

**Comment by lambdamoses** [comment]
Then would you please add that to the documentation? Also, is there a way to preview the tile configurations before deciding whether the flip y? MIST and BigStitcher can do that.

**Comment by lambdamoses** [comment]
That's helpful, thanks!

**Comment by jmuhlich** [comment]
Note that we just released Ashlar v1.13.1 here and on pypi which adds a critical feature for making `preview_slide` usable (again) : downsampling of the image resolution before display. (We previously had a workaround but the key module became obsolete) The default is 10x downsampling but you can adjust that with the `-d` argument if you want more or less downsampling.
